source/databases-collections.txt

```diff
@@ -1,13 +1,137 @@
-Navigate Databases, Collections, and Documents
-==============================================
+==================
+Navigate Your Data
+==================
 
 .. default-domain:: mongodb
 
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
+
+Once you connect to your deployment using |vsce|, you can use the
+left navigation to:
+
+- Explore your databases, collections, and documents. 
+
+- Create new databases and collections.
+
+- Drop databases and collections.
+
+.. figure:: /images/dbs-colls-docs.png
+  :figwidth: 300px
+  :alt: Image showing deployment navigation
+
+Databases and Collections
+-------------------------
+
+When you expand an active connection, |vsce| shows the databases
+in that deployment. Click a database to view the collections that
+database.
+
+Create a New Database
+~~~~~~~~~~~~~~~~~~~~~
+
+To create a new database:
+
+1. Hover over the connection for the deployment where you want your
+   database to exist.
+
+#. Click the appearing :icon-fa5:`plus` icon.
+
+#. In the prompt, enter a name for your new database.
+
+#. Press the enter key.
+
+#. Enter a name for the first collection in your new database.
+
+   .. note::
+
+      You must initialize your new database by populating it with
+      a collection.
+
+#. Press the enter key.
+
+Create a New Collection
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To create a new collection:
+
+1. Hover over the database where you want your collection to exist.
+
+#. Click the appearing :icon:`plus` icon.
+
+#. In the prompt, enter a name for your new collection.
+
+#. Press the enter key to confirm your new collection.
+
+Drop a Database or Collection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important::
+
+   Dropping data from MongoDB is an irreversible process. Be careful to
+   only drop data you are sure you want to delete, and backup your data
+   as necessary.
+
+   Dropping a database also drops all collections and documents within
+   that database.
+
+To drop a database or collection:
+
+1. Right-click the target database or collection.
+
+#. Click :guilabel:`Drop Database` or :guilabel:`Drop Collection`.
+
+#. In the prompt, type the name of the target database or collection.
+
+#. Press the enter key.
+
+View Collection Documents and Schema
+------------------------------------
+
+Click a collection to view its documents and schema.
+
+Documents
+~~~~~~~~~
+
+When you expand a collection's documents, |vsce| lists the ``_id`` of
+each document in the collection. Click an ``_id`` value to open that
+document in VS Code and view its contents.
+
+Alternatively, you can right-click a collection and click
+:guilabel:`View Documents` to view all of the collection's documents
+in an array.
+
+Opening collection documents provides a **read-only** view of your data.
+To modify your data using |vsce|, use a JavaScript Playground.
+
+Schema
+~~~~~~
+
+When you expand a collection's schema, |vsce| lists the fields which
```

Comment:
It sounds like the "schema" display in VSCE is unrelated to JSON schema, which could confuse some users. Do you think we could add an extra sentence to this section explaining this distinction? Not totally sure about the audience of this section so if you think that would be even more confusing we can leave it out.

Reply:
So, it actually is tightly coupled with JSON schema. I'm going to try rephrasing a bit to make the relationship clearer.